sklearn/inspection/_permutation_importance.py

```diff
@@ -1,6 +1,8 @@
 """Permutation importance for estimators."""
+import numbers
 import numpy as np
 from joblib import Parallel
+from sklearn.ensemble._bagging import _generate_indices
```

Comment:
You need to make a relative import

```suggestion
from ..ensemble._bagging import _generate_indices
```

Could you move it to line 6 next to the other relative imports

Reply:
Done